File: Angular/satva/src/app/selltransaction.ts
import { Transaction } from "./transaction";

export class Selltransaction extends Transaction{
    private custId: number = 0;
    public get customerid(): number {
        return this.custId;
    }
    public set customerid(value: number) {
        this.custId = value;
    }
    private rewardPoints: number = 0;
    public get rewardpoints(): number {
        return this.rewardPoints;
    }
    public set rewardpoints(value: number) {
        this.rewardPoints = value;
    }
}

File: SpringBoot/demo/src/main/java/com/example/rps/beans/sell/SellTransaction.java
package com.example.rps.beans.sell;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.List;

import com.example.rps.beans.LocalDateTimeDeserializer;
import com.example.rps.beans.LocalDateTimeSerializer;
import com.example.rps.beans.Paper;
import com.example.rps.beans.Transaction;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
//import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
//import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;

@JsonIgnoreProperties(ignoreUnknown = true)
public class SellTransaction extends Transaction
{
	
	private int custId;	
	private int rewardPoints;
	
	
	public int getRewardPoints() {
		return rewardPoints;
	}
	public void setRewardPoints(int rewardPoints) {
		this.rewardPoints = rewardPoints;
	}
	
	public int getCustId() {
		return custId;
	}
	public void setCustId(int custId) {
		this.custId = custId;
	}
	

}
